browser/resources/brave_new_tab_page_refresh/components/app.style.ts

```diff
@@ -0,0 +1,110 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { color, font } from '@brave/leo/tokens/css/variables'
+import { scoped, global } from '../lib/scoped_css'
+
+export const style = scoped.css`
+  .top-controls {
+    position: absolute;
+    inset-block-start: 4px;
+    inset-inline-end: 4px;
+    min-height: 24px;
+    display: flex;
+    gap: 8px;
+    align-items: center;
+    z-index: 2;
+  }
+
+  .settings {
+    --leo-icon-size: 20px;
+
+    opacity: 0.5;
+    color: #fff;
+    filter: drop-shadow(0px 1px 4px rgba(0, 0, 0, 0.60));
+
+    &:hover {
+      opacity: 0.7;
+      cursor: pointer;
+    }
+  }
+
+  .allow-background-pointer-events {
+    /* This element will allow pointer events to target the background. */
+    pointer-events: none;
+
+    /* But children will not (unless they explicitly allow it). */
+    > :not(.allow-background-pointer-events) {
+      pointer-events: auto;
+    }
+
+    /* And not when a popover is open. When a popover is open, pointer events
+       on a background iframe will not "light-dismiss" the popover. */
+    :scope:has(:popover-open) & {
+      pointer-events: auto;
+    }
+  }
+
+  main {
+    position: relative;
+    z-index: 1;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 100vh;
+    gap: 16px;
+    padding: 16px 24px;
+  }
+
+  .spacer {
+    flex: 1 1 auto;
+    align-self: stretch;
+  }
+`
+
+global.css`
+  @scope (${style.selector}) {
```

Comment:
Sorry, I know you explained this to me, but why do we need a `@scope` on global CSS?

Reply:
Strictly speaking, we don't, but it seems like a good rule that all styles are scoped to some element (even if the rules break through child scopes, as here).

Reply:
I'm okay to leave this, it just reads as slightly confusing - like, a global rule is scoped?

Okay, so next (probably dumb) question is why can't this be part of the `scoped` CSS? Is it because that's only scoped to the next bit of scoped CSS, whereas this will be everything under that selector?